lib/carbon/util.py

```diff
@@ -42,6 +43,25 @@ def dropprivs(user):
   return (uid, gid)
 
 
+def enableTcpKeepAlive(transport):
+  from carbon.conf import settings
+
+  if not settings.TCP_KEEPALIVE or not hasattr(transport, 'getHandle'):
```

Comment:
I'd prefer to see the setting passed in when the function is called, that way it's clearer what's going on.

Reply:
or can I name it `maybeEnableTcpKeepAlive()` and pass it `settings` ? Less code duplication

Reply:
I would prefer to just pass the setting itself, not a big fan of the `maybe` pattern

Reply:
done